validator/coordinator/src/test/java/tech/pegasys/teku/validator/coordinator/DutyMetricsTest.java

```diff
@@ -23,17 +23,24 @@
 import tech.pegasys.teku.infrastructure.metrics.MetricsHistogram;
 import tech.pegasys.teku.infrastructure.time.StubTimeProvider;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.networks.SpecProviderFactory;
+import tech.pegasys.teku.networks.TestConstantsLoader;
+import tech.pegasys.teku.spec.SpecProvider;
+import tech.pegasys.teku.spec.constants.SpecConstants;
 import tech.pegasys.teku.storage.client.RecentChainData;
 
 class DutyMetricsTest {
 
   private static final UInt64 GENESIS_TIME = UInt64.valueOf(100_000);
+  final SpecConstants specConstants =
+      TestConstantsLoader.loadConstantsBuilder("minimal").minGenesisTime(GENESIS_TIME).build();
```

Comment:
Do we actually need to modify minGenesisTime here?

Reply:
still worked without, so removed.